docs/blog/announcing-vite3.md

```diff
@@ -0,0 +1,193 @@
+
+### Vite CLI
+
+<pre style="background-color:rgb(36, 36, 36);padding:2em;">
+  <span style="color:lightgreen"><b>VITE</b></span> <span style="color:lightgreen">v3.0.0</span>  <span style="color:gray">ready in <b>320</b> ms</span>
+
+  <span style="color:lightgreen"><b>➜</b></span>  <span style="color:white"><b>Local</b>:</span>   <span style="color:cyan">http://127.0.0.1:5173/</span>
+  <span style="color:green"><b>➜</b></span>  <span style="color:gray"><b>Network</b>: use --host to expose</span>
+</pre>
+
+Apart from the CLI’s aesthetics improvements, you’ll notice that the default dev server port is now 5173 and the preview server listening at 4173. This change ensures Vite will avoid collisions with other tools.
+
+### Improved WebSocket Connection Strategy
+
+One of the pain points of Vite v2 was configuring the server when running behind a proxy. Vite v3 changes the default connection scheme so it works out of the box in most scenarios. All these setups are now tested as part of the Vite Ecosystem CI through [vite-setup-catalogue](https://github.com/sapphi-red/vite-setup-catalogue).
+
+### ESM SSR build by default
+
+Most SSR frameworks in the ecosystem were already using ESM builds. So, Vite v3 makes ESM the default format for SSR builds. This allows us to streamline previous [SSR externalization heuristics](https://vitejs.dev/guide/ssr.html#ssr-externals). externalizing dependencies by default.
+
+### Cold start improvements
+
+Vite now avoids full reload during cold start when imports are injected by plugins while crawling the initial statically imported modules ([#8869](https://github.com/vitejs/vite/issues/8869)).
+
+In Vite 2.9, both the scanner and optimizer were run in the background. In the best scenario, where the scanner would find every dependency, no reload was needed in cold start. But if the scanner missed a dependency, a new optimization phase and then a reload were needed. Vite was able to avoid some of these reloads in v2.9, as we detected if the new optimized chunks were compatible with the ones the browser had. But if there was a common dep, the sub-chunks could change and a reload was required to avoid duplicated state. In Vite 3, the optimized deps aren't handed to the browser until the crawling of static imports is done. A quick optimization phase is issued if there is a missing dep (for example, injected by a plugin), and only then, the bundled deps are sent. So, a page reload is no longer needed for these cases.
+
+<img style="background-color:rgb(36, 36, 36);padding:5%;" width="90%" height="auto" src="../images/vite-3-cold-start.svg">
```

Comment:
I feel this figure a bit small. Could the text be larger? or if it is intended to be small, maybe we could make it able to zoom it by clicking the image.
![image](https://user-images.githubusercontent.com/49056869/178412647-bafef68a-e50d-48fe-91fa-9c4983fa3702.png)

Reply:
Ya, I'll check if I can get the original and increase the font 👍🏼

Reply:
done, let me know if it better

Reply:
looks better to me 👍